README.md

```diff
@@ -5,6 +5,7 @@ The intention of this bot is to make moderation easier and provide useful comman
 
 ## Running the bot
 1. Copy `src/main/resources/sample-config.yml` and use it to structure your bot config.
+    We recommend to copy the file to ./local-config.yml since this is added to the .gitignore file so that your token is not pushed to the repository. 
```

Comment:
Explaining things with simplicity but also specifics is important for our README so that new programmers can jump in and not be intimidated by any verbiage. Maybe something like 

> We recommend to copy the file as `local-config.yml` to the project base directory so that your bot token is not pushed to the repository.

would be more beneficial as it keeps it simple but also tries to be detailed in instruction. Renaming the filename to just `config.yml` may also be something to keep in mind. Regardless, thank you for taking the time to make a pull request!